qubes-host.repo.in

```diff
@@ -1,41 +1,41 @@
-[qubes-dom0-current]
-name = Qubes Dom0 Repository (updates)
-#baseurl = https://yum.qubes-os.org/r$releasever/current/dom0/%DIST%
-#baseurl = http://yum.qubesosfasa4zl44o4tws22di6kepyzfeqv3tg4e3ztknltfxqrymdad.onion/r$releasever/current/dom0/%DIST%
-metalink = https://yum.qubes-os.org/r$releasever/current/dom0/%DIST%/repodata/repomd.xml.metalink
+[qubes-host-current]
```

Comment:
URL ok, but name of the repository need to stay old, too many places have references to it, including documentation, various scripts etc.